scripts/qemu/parse_config.py

```diff
@@ -50,6 +53,9 @@ def parse_topology_xml(tpg_file_name: str) -> TopologyCfg:
         result.check_returncode()
         libvirt_args = result.stdout.decode("utf-8").strip()
 
+        if verbose_mode != False:
```

Comment:
if verbose_mode:

Reply:
I know this should work, but:
- I followed the same pattern as in other places here,
- plus, we just want to make sure it's any value, but not the default one (`os.getenv("ENABLE_VERBOSE", False)`)

If you want I can change it in all places, but I'd personally leave it as is.